docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc

```diff
@@ -1,17 +1,34 @@
-=Jenkins Remoting Monitoring
-*Goal:*  Support monitoring of Jenkins remoting networking with open source monitoring tools such as Prometheus, Grafana, etc."
-s, Core
+= Jenkins Remoting Monitoring 
 
+*Project goal:* Support monitoring of Jenkins remoting networking with open source monitoring tools such as Prometheus, Grafana, etc.
 
+*Skills to study/improve:* Java, Networking, Docker, Prometheus/Grafana/etc.
 
-project: remoting
+This project idea is published as a Google doc. You are welcome to comment on the proposal or to suggest changes in the draft embedded below ( https://docs.google.com/document/d/1nhb-lfdjxT2fAY3US15HIMvePQCCoWU82VV1Y8Mu3lc[Google Doc] )
 
+++++
+<iframe src="https://docs.google.com/document/d/1nhb-lfdjxT2fAY3US15HIMvePQCCoWU82VV1Y8Mu3lc" width="100%" height="600px"></iframe>
+++++
 
-# To be confirmed
-#- "linuxsuren"
-#- "mikecirioli"
-#- "markyjackson-taulia"
```

Comment:
Why are these deleted?

Reply:
These are comments. If we want them. I can add them back.

Reply:
These are not just comments, instead they are comments of historical significance. Please add these back.